implementations/ApiCatalog.json

```diff
@@ -1,38 +1,95 @@
 {
-  "name": "apicatalog.com",
-  "implementation": "Iron Verifiable Credentials",
-  "verifiers": [{
-    "id": "https://vc.apicatalog.com",
-    "endpoint": "https://vc.apicatalog.com/credentials/verify",
-    "supportedEcdsaKeyTypes": ["P-256", "P-384"],
-    "tags": ["Ed25519Signature2020", "vc2.0", "eddsa-rdfc-2022", "ecdsa-rdfc-2019"]
-  }],
-  "issuers": [{
-    "id": "did:key:z6Mkska8oQD7QQQWxqa7L5ai4mH98HfAdSwomPFYKuqNyE2y",
-    "endpoint": "https://vc.apicatalog.com/credentials/issue",
-    "options": {
-      "type": "Ed25519Signature2020"
-    },
-    "tags": ["Ed25519Signature2020", "vc2.0"]
-  }, {
-    "id": "did:key:z6Mkska8oQD7QQQWxqa7L5ai4mH98HfAdSwomPFYKuqNyE2y",
-    "endpoint": "https://vc.apicatalog.com/credentials/issue",
-    "options": {
-      "type": "eddsa-2022"
-    },
-    "tags": ["eddsa-rdfc-2022"]
-  }, {
-    "id": "did:key:zDnaepBuvsQ8cpsWrVKw8fbpGpvPeNSjVPTWoq6cRqaYzBKVP",
-    "endpoint": "https://vc.apicatalog.com/credentials/issue",
-    "options": {
-      "type": "ecdsa-2019"
-    },
-    "supportedEcdsaKeyTypes": ["P-256", "P-384"],
-    "tags": ["ecdsa-rdfc-2019"]
-  }],
-  "vpVerifiers": [{
-    "id": "https://vc.apicatalog.com",
-    "endpoint": "https://vc.apicatalog.com/presentations/verify",
-    "tags": ["Ed25519Signature2020", "vc2.0", "eddsa-rdfc-2022", "ecdsa-rdfc-2019"]
-  }]
+	"name": "apicatalog.com",
```

Comment:
Tabs?! 😜 

We should get an `.editorconfig` added here, so the code editors/UIs can do this for everyone.

@filip26 can you change these to two spaces?

Reply:
@BigBlueHat sorry, sure, It's Eclipse, I thought I've only spaces in settings.